README.md

```diff
@@ -10,6 +10,8 @@ _**For more information take a look at the [issues](https://github.com/tomsom/yo
 
 ## Latest Fixes:
 > **Kernel independent:**
+ > - [x] **Fingerprint Sensor**
+ > - Seems to be working with the driver from this [Copr repo](https://copr.fedorainfracloud.org/coprs/d-k-bo/libfprint-goodixtls/).
```

Comment:
This is still heavy WIP, not stable, and not cross platform. Should probably be put into a "Workaround" section to show the significance of not being a proper fix yet.